src/Player.js

```diff
@@ -17,6 +17,9 @@ class Player {
         const data = JSON.parse(msg);
         if (!data.type) {
             return;
+        } else if (data.type === 'req') {
```

Comment:
This can be removed. Leftover from when I was experimenting with the client requesting frames instead of broadcasting